src/api/contextMenu/ContextMenuAPI.js

```diff
@@ -75,9 +75,11 @@ class ContextMenuAPI {
     /**
      * @private
      */
-    _showContextMenuForObjectPath(objectPath, x, y, actionsToBeIncluded) {
-
-        let applicableActions = this._allActions.filter((action) => {
+    _showContextMenuForObjectPath(objectPath, x, y, actionsToBeIncluded, actionsToBeSkipped = []) {
```

Comment:
lets change this to an all inclusive options object, that would have a actions to be included and an actions to be skipped. Keeping the arguments from increasing too much